openrr-teleop/Cargo.toml

```diff
@@ -28,4 +28,5 @@ tokio = { version = "1.0", features = ["rt-multi-thread", "sync", "time"] }
 tracing = { version = "0.1", features = ["log"] }
 
 [dev-dependencies]
+approx = "0.5.0"
```

Comment:
If you have no reason to use approx instead of assert_approx_eq, please use assert_approx_eq to keep consistency and not to increase dependencies.

Reply:
ok.